test/EndToEnd-Installer.Tests/GivenWindowsApp.cs

```diff
@@ -19,6 +19,7 @@ public class GivenWindowsApp : TestBase
         [InlineData("10.0.20348.0")]
         [InlineData("10.0.22000.0")]
         [InlineData("10.0.22621.0")]
+        [InlineData("10.0.26100.0")]
```

Comment:
@manodasanW You added this to a test that wasn't running yet. I just got these tests [running here](https://github.com/dotnet/sdk/pull/42195) and this new version fails against the [Microsoft.NET.TargetFrameworkInference.targets](https://github.com/dotnet/sdk/blob/main/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets) stating:

```
"C:\h\w\9909085F\t\dotnetSdkTests\bsxetacl.wpv\ItCanBuildAnd---57E62695\consolecswinrt.csproj" (Build target) (1:7) -> (_CheckForInvalidTargetPlatformVersion target) ->
C:\h\w\9909085F\p\d\sdk\9.0.100-ci\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.TargetFrameworkInference.targets(259,5):
  error NETSDK1140: 10.0.26100.0 is not a valid TargetPlatformVersion for Windows. Valid versions include: [C:\h\w\9909085F\t\dotnetSdkTests\bsxetacl.wpv\ItCanBuildAnd---57E62695\consolecswinrt.csproj]
10.0.22621.0
10.0.22000.0
10.0.20348.0
10.0.19041.0
10.0.18362.0
10.0.17763.0
10.0.22000.0
10.0.20348.0
10.0.19041.0
10.0.18362.0
10.0.17763.0
8.0
7.0
```

Is something missing from `Microsoft.NET.TargetFrameworkInference.targets`?

Reply:
That is strange.  Is somehow the change in `src/Installer/redist-installer/targets/GenerateBundledVersions.targets` not getting picked up?  When I did this change, I had manually edited my local .NET SDK installation and it did seem to work in a project (didn't try the test itself).

Reply:
Ohhhhhh. Huh. I don't think the test is running against any outputs of `redist-installer`. It is running against the same outputs of the SDK build. We don't have a separate test process that runs against `redist-installer` outputs. I think this would have to wait until the entire redist process is merged into one process.

Reply:
I talked to @marcpopMSFT and now I understand the situation. The version of the SDK specified in `global.json` wasn't updated yet to have the `GenerateBundledVersions.targets` changes. This updates the "stage 0" SDK, which is what the tests are running against. Therefore, the test fails because your change isn't there yet. As mentioned, until we merge the `redist` and `redist-installer` process into one, this situation will happen. It is on our plate to do. For now, either we wait until Arcade updates our `global.json`, or we could make a PR that manually updates it and uncomments the InlineData node. Makes sense?

Reply:
Thanks for figuring this out.  Yea I am fine with either approach.  I assume by the time .NET 9 ships, the SDK will be updated to use the updated redist-installer right?

Reply:
Correct, by default we'll be updated to a new SDK through automatic arcade updates but we'll have to reenable the test them. Eventually we'll merge the two redists as mentioned above this won't be an issue anymore after that. No eta for that though as it's complicated.